packages/@react-spectrum/list/test/ListView.test.js

```diff
@@ -90,6 +90,8 @@ describe('ListView', function () {
     );
     // Allow for Virtualizer layout to update
     act(() => {jest.runAllTimers();});
+    // Allow section layout info to be updated as a result of the initial render's useVirtualizerItem call
+    act(() => {jest.runAllTimers();});
```

Comment:
This is actually something that we are missing in our tests in main for ListBox, we just haven't encountered it because we don't have PageUp/Down tests. useVirtualizerItem updates the section header height in a useLayoutEffect to match the test's mocked scrollHeight from the default of 48 which ListLayout sets initially. This `virtualizer.updateItemSize` call triggers a raf [here](https://github.com/adobe/react-spectrum/blob/93c6b3c17e205df552f3295f909581367ca53912/packages/%40react-stately/virtualizer/src/Virtualizer.ts#L463-L466), hence the need for a second runAllTimers